blog/2024-06-18-async_register_static_path.md

```diff
@@ -0,0 +1,32 @@
+---
+author: J. Nick Koston
+authorURL: https://github.com/bdraco
+title: Making http path registration async safe with `async_register_static_path`
+---
+
+`hass.http.register_static_path` is deprecated because it does blocking I/O in the event loop, instead call `await hass.http.async_register_static_path([StaticPathConfig(url_path, path, cache_headers)])`
+
+The arguments to `async_register_static_path` are the same as `register_static_path` except they are wrapped in the `StaticPathConfig` `dataclass` and an `Iterable` of them is accepted to allow registering multiple paths at once to avoid multiple executor jobs.
```

Comment:
```suggestion
The arguments to `async_register_static_paths` are the same as `register_static_path` except they are wrapped in the `StaticPathConfig` `dataclass` and an `Iterable` of them is accepted to allow registering multiple paths at once to avoid multiple executor jobs.
```